hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -419,5 +419,30 @@
       stage: V2
       tests: ["i2c_target_stretch"]
     }
+    {
+      name: bad_address
+      desc: '''
+            Test sends transactions with a randomized address including two legal (programmed) addresses.
+            Run this test with dut target mode.
+
+            Checking:
+            All transactions with illegal addresses should be dropped silently and should not disturb
+            transactions with legal address
+            '''
+      stage: V2
+      tests: ["i2c_target_bad_addr"]
+    }
+    {
+      name: runt_frame
```

Comment:
what does `runt` mean?

Reply:
If we define a frame all byte between 'start' and 'stop', runt means
a frame without either 'start' or 'stop'.

Reply:
I see. Thanks for the explanation.